Code/Editor/IEditorImpl.cpp

```diff
@@ -1460,7 +1460,7 @@ void CEditorImpl::UnregisterNotifyListener(IEditorNotifyListener* listener)
 
 ISourceControl* CEditorImpl::GetSourceControl()
 {
-    CryAutoLock<CryMutex> lock(m_pluginMutex);
+    AZStd::lock_guard<AZStd::mutex> lock(m_pluginMutex);
```

Comment:
Recommendation: The lock_guard can be replaced with a CTAD scoped lock

```suggestion
    AZStd::scoped_lock lock(m_pluginMutex);
```